composer.json

```diff
@@ -21,10 +21,6 @@
 		"symfony/console": "3.4.16 || 4.1.5"
 	},
 	"require-dev": {
-		"ext-gd": "*",
-		"ext-intl": "*",
-		"ext-mysqli": "*",
-		"ext-zip": "*",
```

Comment:
:disappointed: in this way, when I fetch phpstan and want to contribute, I need to find out for myself (test are failing, for example: `Class ZipArchive was not found while trying to analyse it`)

Reply:
Probably `composer install --ignore-platform-reqs` flag will help you

Reply:
Nope, with this extensions tests are failing.

Reply:
I add fix for it in #1731